app/views/schools/participants/show.html.erb

```diff
@@ -2,17 +2,16 @@
 
 <% content_for :before_content, govuk_back_link( text: "Back", href: schools_participants_path) %>
 
-<span class="govuk-caption-xl"><%= @school.name %></span>
-<h1 class="govuk-heading-xl"><%= @profile.user.full_name %></h1>
-
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-m">Participant details</h2>
+    <div class="govuk-grid-column-two-thirds">
+        
+        <span class="govuk-caption-xl"><%= @school.name %></span>
+        <h1 class="govuk-heading-xl"><%= @profile.user.full_name %></h1>        
 
-    <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+        <dl class="govuk-summary-list govuk-!-margin-bottom-7">
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
-          Full name
+          Name
```

Comment:
Indentation in this one has gone a bit squiffy

Reply:
Fixed - pushing up